docs/install.md

```diff
@@ -162,6 +201,59 @@ Select the following tabs for the installation description of your target.
 
         Then exit `dietpi-config`. After this the first time installer procedure should run again from the start.
 
+    ??? important "Installation steps within the DietPi guest system when using the VirtualBox extension pack"
+        If you use the VirtualBox extension pack, after the DietPi base installation (done during the very first boot of the DietPi system), further installation steps within the DietPi guest system have to be done to achieve a working time synchronization. In general you have to use the following **Time sync mode** options set via the `dietpi-config` command (in the **Advanced Options**):
+
+        - VirtualBox without extension pack: The time synchronisation has to use options 1..4 ("Boot only", "Boot + Daily", "Boot + Hourly", "Daemon + Drift"). No further installation steps are necessary.
+        - VirtualBox with extension pack: The time synchronisation must use option 0 ("Custom"). Further installation steps are necessary, see below.
+
+            ![DietPi-config time synchronization](assets/images/dietpi-config-timesync.png){: width="600" height="328" loading="lazy"}
+
+        Additional installation steps in case of the use of the extension pack:
+
+        1. Check whether an optical disc with the VirtualBox extensions (`.iso`) is present:
+
+            ```sh
+            lsblk
+            ```
+
+            If you do not see a `/dev/sr0` optical drive, you need to add this in the virtual machine settings (in the shutdown state of the VM) `\Machine->Settings->Storage`: Add an optical storage which is linked to the guest additions (`.iso`).
+
+        2. Install the kernel headers and the extension pack:
+
+            ```sh
+            apt -y install build-essential dkms linux-headers-amd64
+            cd ~
```

Comment:
`cd ~` has no effect here, as we navigate into the mount point below.

Reply:
Ok, thx.